navigation/ca_move_base/launch/move_base.launch

```diff
@@ -16,7 +16,7 @@
     <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
     <rosparam file="$(find ca_move_base)/config/local_costmap_params.yaml" command="load" />
     <rosparam file="$(find ca_move_base)/config/$(arg local_planner)_local_planner_params.yaml" command="load" />
-
+    <param name="SBPLLatticePlanner/primitive_filename" value="$(find ca_move_base)/config/matlab/turtle.mprim" />
```

Comment:
Can you move it to line 43 close to the TEB parameter?

Reply:
Addressed